### PR TITLE
deprecate bintray

### PR DESCRIPTION
**Pull request by alita-moore**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -56,7 +56,7 @@ export RABBITMQ_PLUGINS_DIR='/app/.apt/usr/lib/rabbitmq/lib/rabbitmq_server-3.6.
 set-env RABBITMQ_ENABLED_PLUGINS_FILE '$RABBIT_TMP/plugins/enabled_plugins'
 export RABBITMQ_ENABLED_PLUGINS_FILE="$RABBIT_TMP/plugins/enabled_plugins"
 touch $RABBITMQ_ENABLED_PLUGINS_FILE
-wget https://dl.bintray.com/rabbitmq/community-plugins/3.6.x/rabbitmq_delayed_message_exchange/rabbitmq_delayed_message_exchange-20171215-3.6.x.zip -P /app/.apt/usr/lib/rabbitmq/lib/rabbitmq_server-3.6.10/plugins
+wget https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/archive/refs/tags/rabbitmq_v3_6_10.zip -P /app/.apt/usr/lib/rabbitmq/lib/rabbitmq_server-3.6.10/plugins
 cd /app/.apt/usr/lib/rabbitmq/lib/rabbitmq_server-3.6.10/plugins
 unzip rabbitmq_delayed_message_exchange-20171215-3.6.x.zip
```
Comment (aduca98) [inline bin/compile:61]: i think you'll have to change this line too etc... as some of the lines are tied to the old zipfiles name. ex. `rabbitmq_delayed_message_exchange-20171215-3.6.x.zip` needs to change to `rabbitmq_v3_6_10.zip`.

also how should we go about testing this?

**Review by aduca98 — CHANGES_REQUESTED** [review]

File: bin/compile (hunk 1 above)
Comment (alita-moore) [reply bin/compile:61]: we can use a build back release on staging

Comment (alita-moore) [reply bin/compile:61]: Also, I'm curious though because it really shouldn't matter

Comment (alita-moore) [reply bin/compile:61]: i.e. in both cases it saves to `rabbitmq_server3.6.10/plugins` so you'd think it'd all be the same

Comment (alita-moore) [reply bin/compile:61]: for the testing though go to heroku -> staging -> settings -> buildpacks

**Comment by alita-moore** [comment]
this only impacts CI so merging anyway

**Pull request merged**